d3a_api_client/utils.py

```diff
@@ -6,11 +6,13 @@
 import logging
 import uuid
 from functools import wraps
-from tabulate import tabulate
+
 from d3a_interface.utils import key_in_dict_and_not_none, get_area_name_uuid_mapping,RepeatingTimer
 from d3a_interface.constants_limits import JWT_TOKEN_EXPIRY_IN_SECS
 from d3a_api_client.constants import DEFAULT_DOMAIN_NAME, DEFAULT_WEBSOCKET_DOMAIN
 
+from tabulate import tabulate
+
```

Comment:
Please revert to follow PEP8

Reply:
Yeah, I don't know why I thought it was in the wrong place :D Fixed that and the other imports' sorting.